src/modules/web3/model/funds-deposited-ev.entity.ts

```diff
@@ -0,0 +1,48 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, Unique, UpdateDateColumn } from "typeorm";
+
+export type FundsDepositedEventArgs = {
```

Comment:
NIT: Can't we get this from the typechain bindings in `@across-protocol/contracts-v2`?